### PR TITLE
List blog articles and categories

### PR DESCRIPTION
**Pull request by gustaveen**
## What?

- Add `ArticleListBlock` to list blog posts that takes category as an optional parameter. 
- Add `category` content type to create a taxonomy for articles and also function as a page to list articles of that category
- Add `storyblok/react` and bump node to make it work
- Add new type for the latest Storyblok `Image`

Sorry for the big PR 🙃

Next steps:
- Add category navigation
- Add "More articles" button to load more articles

## Why?
We're gonna list articles on `/blog` and have specific category pages like `/blog/press` and `/blog/news`. 

**Ticket(s): []**

### /blog
Screenshot 2023-01-17 at 17 32 20

### /[category]
Screenshot 2023-01-17 at 17 32 57

### Article
Screenshot 2023-01-17 at 17 33 11

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* master
  * **PR #701** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-next/701" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-next/701?utm_source=stack-comment).

File: src/blocks/ArticleListBlock/ArticleTeaser.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    top: 0;
+    left: 0;
+  }
+`
+
+const Title = styled.h3`
+  font-size: 1.5rem;
+  line-height: 1.2;
+
+  && {
```
Comment (gustaveen) [inline src/blocks/ArticleListBlock/ArticleTeaser.tsx:35]: Alternative to `!important` since I needed to increase specificity :)

**Review by robinandeer — APPROVED** [review]
Looks good to me - just some small comments

[happynic](https://app.graphite.dev/macros/HedvigInsurance)

File: src/blocks/ArticleListBlock/ArticleListBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+export const ArticleListBlock = ({ category }: ArticleListBlockProps) => {
+  const {
+    currentLocale: { label },
+  } = useLocale()
+  const [articles, setArticles] = useState([])
+
+  useEffect(() => {
+    const getArticles = async () => {
+      const storyblokApi = getStoryblokApi()
+      const { data } = await storyblokApi.get(`cdn/stories/`, {
```
[the remark this answers is not in the document]
Comment (gustaveen) [reply src/blocks/ArticleListBlock/ArticleListBlock.tsx:23]: Hear hear! Had to see through the fingers on this one haha. The overhead of making this work through the server/koa setup was to great for this 🤪

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-v14.17.6
+v16.18.1
```
[the remark this answers is not in the document]
Comment (gustaveen) [reply .nvmrc:1]: I was a bit cautious to good ol' market-web 😆 But let's try and bump it even further! 👌 @robinandeer I hold that of to a new PR

**Pull request merged**